develop/dev-guide-third-party-tools-compatibility.md

```diff
@@ -13,7 +13,6 @@ summary: 介绍在测试中发现的 TiDB 与第三方工具的兼容性问题
 > - 触发器
 > - 事件
 > - 自定义函数
-> - 外键约束
```

Comment:
- Remove Line 165

  ```
  - 不支持与外键约束相关的功能（包括多对多关联）。
  ```
- Modify the FAQ [1.1.6](https://docs.pingcap.com/zh/tidb/dev/tidb-faq#116-tidb-%E5%92%8C-mysql-%E5%85%BC%E5%AE%B9%E6%80%A7%E5%A6%82%E4%BD%95)

  ```
  TiDB 支持绝大部分 MySQL 5.7 的语法，但目前还不支持触发器、存储过程、自定义函数、外键约束等。详情参见[与 MySQL 兼容性对比](/mysql-compatibility.md)。
  ```